assets/js/calypsoify.js

```diff
@@ -1,363 +1,371 @@
-            icon = icons.notice;
-        }
-        $( this ).prepend( '<span class="wc-calypso-bridge-notice-icon-wrapper">' + icon + '</span>' );
-    } );
-
-    /**
-     * Replace dismissal buttons in notices
-     */
-    $( document ).ready( function() {
-        $( '.notice-dismiss' ).html( icons.cross );
-    } );
-
-    /**
-     * Place notice content inside it's own tag
-     * 
-     * Used to prevent side by side content in flexbox when multiple paragraphs exist.
-     */
-    $( 'div.notice, div.error, div.updated, div.warning' ).each( function() {
-        var $noticeContent = $( '<div class="wc-calypso-bridge-notice-content"></div>' );
-        $( this ).find( '.wc-calypso-bridge-notice-icon-wrapper' ).after( $noticeContent );
-        $( this ).find( 'p:not(.submit)' ).appendTo( $noticeContent );
-    } );
-
-    /**
-     * Move page actions to action header
-     */
-    $( '.page-title-action, .add-new-h2' ).appendTo( '#action-header .action-header__actions' );
-
-    /** 
-     * Move notices on pages with sub navigation
-     * 
-     * WP Core moves notices with jQuery so this is needed to move them again since
-     * we can't control their position.
-     */
-    $( document ).ready(function() {
-        var $subNavigation = $( '.wrap .subsubsub' );
-        if ( $subNavigation.length ) {
-			$( 'div.notice, div.error, div.updated, div.warning' ).insertAfter( $subNavigation.first() );
+	'use strict';
+
+	/**
+	 * Action header mobile navigation.
+	 */
+	$( document ).on( 'click', '.action-header:not(.action-header-sidebar) .action-header__ground-control-back', function( e ) {
+		if ( $( window ).width() < 661 ) {
+			e.preventDefault();
+			$( '#wp-admin-bar-menu-toggle .ab-item' ).click();
+		}
+	} );
+
+// @todo Remove the notice handlers after https://github.com/Automattic/jetpack/pull/13126 has been released.
+	/**
+	 * Prepend icons to notices.
+	 */
+	$( 'div.notice, div.error, div.updated, div.warning' ).each( function() {
+		if ( $( this ).children( '.wc-calypso-bridge-notice-content' ).length ) {
+			return;
+		}
+
+		var icon = icons.info;
+		if ( $( this ).hasClass( 'notice-success' ) ) {
+			icon = icons.checkmark;
+		} else if ( $( this ).hasClass( 'error' ) || $( this ).hasClass( 'notice-warning' ) ) {
+			icon = icons.notice;
+		}
+		$( this ).prepend( '<span class="wc-calypso-bridge-notice-icon-wrapper">' + icon + '</span>' );
+	} );
+
+	/**
+	 * Replace dismissal buttons in notices.
+	 */
+	$( document ).ready( function() {
+		$( '.notice-dismiss' ).html( icons.cross );
+	} );
+
+	/**
+	 * Place notice content inside it's own tag.
+	 *
+	 * Used to prevent side by side content in flexbox when multiple paragraphs exist.
+	 */
+	$( 'div.notice, div.error, div.updated, div.warning' ).each( function() {
+		if ( $( this ).children( '.wc-calypso-bridge-notice-content' ).length ) {
+			return;
+		}
+
+	var $noticeContent = $( '<div class="wc-calypso-bridge-notice-content"></div>' );
```

Comment:
Small indentation fix needed here.

Reply:
Fixed in https://github.com/Automattic/wc-calypso-bridge/pull/491/commits/70e7e1e94264fb6553d259b0eac830c8c71a98b6